### PR TITLE
please correct me!

### PR DESCRIPTION
**Pull request by llueder**
this is a very well explanation of my change.

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+tis
```
Comment (C0nst21) [inline file.txt:1]: this

**Comment by llueder** [comment]
I'm really not sure about this change. May I have some opinions?

**Comment by talgat-alimbayev** [comment]
correct the spelling errors please

File: file.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+iss
```
Comment (C0nst21) [inline file.txt:2]: is

File: file.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+verri
```
Comment (C0nst21) [inline file.txt:3]: very

File: file.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+bed
```
Comment (C0nst21) [inline file.txt:4]: bad

**Review by andreiraz — CHANGES_REQUESTED** [review]
heey, can you give us a more detailed review of your change? :)

File: file.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+enklish.
```
Comment (C0nst21) [inline file.txt:5]: english

**Review by C0nst21 — COMMENTED** [review]
You should fix that!

**Comment by Odkane** [comment]
not that good

File: file.txt (hunk 1 above)
Comment (talgat-alimbayev) [inline file.txt:1]: for example this line is wrong

Comment (llueder) [inline file.txt:1]:
```suggestion
tis
```

File: file.txt (hunk 2 above)
Comment (talgat-alimbayev) [inline file.txt:2]: and this line as well

Comment (llueder) [reply file.txt:2]: yep, your right, didnÄ't see that

File: file.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+this
+iss
```
Comment (adithyakumarcr) [inline file.txt:2]: could change that

File: file.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+this
```
Comment (Odkane) [inline file.txt:1]: you can do this better ;-)

**Review by andreiraz — APPROVED** [review]
well done!

**Pull request merged**